draft-ietf-quic-transport.md

```diff
@@ -2669,7 +2671,34 @@ the same static key (see {{reset-oracle}}).  A connection ID from a connection
 that is reset by revealing the Stateless Reset Token cannot be reused for new
 connections at nodes that share a static key.
 
-Note that Stateless Reset messages do not have any cryptographic protection.
+Note that Stateless Reset packets do not have any cryptographic protection.
+
+
+#### Looping {#reset-looping}
+
+The design of a Stateless Reset is such that it is indistinguishable from a
+valid packet.  This means that a Stateless Reset might trigger the sending of a
+Stateless Reset in response, which could lead to infinite exchanges.  An
+endpoint MUST use any one of the following measures to limit the sending of
+Stateless Reset:
+
+* An endpoint can remember the number of Stateless Reset packets that it has
+  sent and stop generating new Stateless Reset packets once a limit is reached.
+  Using separate limits for different remote addresses - or different values for
+  the Destination Connection ID fields on the packet that triggers the reset -
+  will ensure that Stateless Reset packets can be used to close connections when
+  other peers or connections have exhausted limits.
+
+* An endpoint can ensure that every Stateless Reset that it sends is smaller
```

Comment:
I think it's worth advising that a stateless reset should never be larger than the received packet.  This makes this defense unilaterally useful, because at least one side is reducing the size every time.

Reply:
That's a good point, and a reason that the TTL defense was removed.  I will restructure.

Reply:
On second thought: if one peer is remembering, then the problem does eventually go away without size reductions.  But it requires that the state is not bound to a connection ID (as @kazuho suggested), it has to be bound to a remote address.

I have restructed the text to emphasize size reduction as the first defense.   State is then an alternative.